components/forms/conditionOfJoining/CojView.tsx

```diff
@@ -27,22 +31,125 @@ function CojDeclarationSection() {
+              type="checkbox"
+              name="isDeclareProvisional"
+              items={[
+                {
+                  label: `I understand that programme and post allocations are provisional and subject to change until confirmed by HEE WM and/or my employing organisation.`,
+                  value: true
+                }
+              ]}
+            />
+            <MultiChoiceInputField
+              id="isDeclareSatisfy"
+              type="checkbox"
+              name="isDeclareSatisfy"
+              items={[
+                {
+                  label:
+                    "I understand that I will need to satisfy all requirements of the programme and curriculum to enable satisfactory sign off, and that this may require a specific time commitment.",
+                  value: true
+                }
+              ]}
+            />
+            <MultiChoiceInputField
+              id="isDeclareProvide"
+              type="checkbox"
+              name="isDeclareProvide"
+              items={[
+                {
+                  label: `I will obtain and provide my School and HEE WM with a professional email address.`,
+                  value: true
+                }
+              ]}
+            />
+            <MultiChoiceInputField
+              id="isDeclareInform"
+              type="checkbox"
+              name="isDeclareInform"
+              items={[
+                {
+                  label: `I will inform my School and HEE WM of any change of my personal contact details and/or personal circumstances that may affect my training programme arrangements.`,
+                  value: true
+                }
+              ]}
+            />
+            <MultiChoiceInputField
+              id="isDeclareUpToDate"
+              type="checkbox"
+              name="isDeclareUpToDate"
+              items={[
+                {
+                  label: `I will keep myself up to date with the latest information available via HEE as well as via the relevant educational and regulatory websites.`,
+                  value: true
+                }
+              ]}
+            />
+            <MultiChoiceInputField
+              id="isDeclareAttend"
+              type="checkbox"
+              name="isDeclareAttend"
+              items={[
+                {
+                  label: `I will attend the minimum number of formal teaching days as required by my School/programme.`,
+                  value: true
+                }
+              ]}
+            />
+            <MultiChoiceInputField
+              id="isDeclareEngage"
+              type="checkbox"
+              name="isDeclareEngage"
+              items={[
+                {
+                  label: `Where applicable, I will fully engage with immigration and employer requirements relating to Tier 2 and Tier 4 UK visas.`,
+                  value: true
+                }
+              ]}
+            />
```

Comment:
@DorisWongNHS , the implementation looks good. One thing you might want to do is refactor this component (following DRY principle). e.g. make an array of checkboxProps objects (key/value pairs of id, label props) which can be imported from the constants file), then loop through this array to render a MultiChoiceInputField for each item in the array.